torchreid/models/common.py

```diff
@@ -60,6 +60,19 @@ def get_num_features(self):
     def get_num_head_features(self):
         return self.num_head_features
 
+    def get_config_optim(self, lrs):
```

Comment:
Why do we need two copies of this method here?

Reply:
Sorry, found it. I will remove this.